website/content/docs/other-specifications/acl-policy.mdx

```diff
@@ -0,0 +1,350 @@
+  as CSI plugins.
+- `csi-write-volume` - Allows CSI volumes to be registered or deregistered.
+- `csi-read-volume` - Allows inspecting a CSI volume and seeing fine grain
+  status.
+- `csi-list-volume` - Allows listing CSI volumes and seeing coarse grain status.
+- `csi-mount-volume` - Allows jobs to be submitted that claim a CSI volume.
+- `list-scaling-policies` - Allows listing scaling policies.
+- `read-scaling-policy` - Allows inspecting a scaling policy.
+- `read-job-scaling` - Allows inspecting the current scaling of a job.
+- `scale-job`: Allows scaling a job up or down.
+- `sentinel-override` - Allows soft mandatory policies to be overridden.
+
+The coarse-grained policy permissions are shorthand for the following fine-
+grained namespace capabilities:
+
+<!-- markdownlint-disable -->
+
+| Policy  | Capabilities                                                                                                                                                                                                                                                    |
+| ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `deny`  | deny                                                                                                                                                                                                                                                            |
+| `read`  | list-jobs<br />parse-job<br />read-job<br />csi-list-volume<br />csi-read-volume<br />list-scaling-policies<br />read-scaling-policy<br />read-job-scaling                                                                                                      |
+| `write` | list-jobs<br />parse-job<br />read-job<br />submit-job<br />dispatch-job<br />read-logs<br />read-fs<br />alloc-exec<br />alloc-lifecycle<br />csi-write-volume<br />csi-mount-volume<br />list-scaling-policies<br />read-scaling-policy<br />read-job-scaling<br />scale-job |
+| `scale` | list-scaling-policies<br />read-scaling-policy<br />read-job-scaling<br />scale-job                                                                                                                                                                             |
+
+<!-- markdownlint-enable -->
+
+If you provide both a `policy` and `capabilities` list, the capabilities are
+merged. For example, the policy below adds the submit-job capability to the read
+policy disposition, which provides the list-job and read-job capabilities:
+
+```hcl
+# Allow reading jobs and submitting jobs, without allowing access
+# to view log output or inspect the filesystem
+namespace "default" {
+  policy       = "read"
+  capabilities = ["submit-job"]
+}
+```
+
+This same policy could also be expressed as:
```

Comment:
```suggestion
A similar policy could also be expressed as:
```

Nit-picking: the `read` disposition got some new values from when this was written. Keeping this updated so the policies are exactly the same would be annoying 😅